setup.py

```diff
@@ -50,6 +50,7 @@ def find_scripts(script_dir, prefix):
         'scipy',
         'seaborn',
         'tables',
+        'joblib',
```

Comment:
`joblib` is included in the ctapipe environment installation, I do not think we need it here, do we?

Reply:
Rigth, i did not notice that

Reply:
No, this is not how this works.

Please list all *direct* dependencies of lstchain in `install_requires`. Do not rely on transitive depdendencies through your dependencies.

Reply:
If you import a package, it goes into `install_requires`

Reply:
That makes sense, I will add it back